### PR TITLE
Group plan owners report

### PR DESCRIPTION
**Pull request by SabreCat**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Your_Local_Install_to_Modify_Habitica%27s_Website_and_API for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)

I've been meaning to start adding ad hoc reports to the repo for a while--this is me starting to do so in a formal fashion.

At present, this is a Mongo shell script, which doesn't meet linting standards. Would it be better to refactor this so it's run via Node? @TheHollidayInn 

[//]: # (Put User ID in here - found on the Habitica website at User Icon > Settings > API)

### CODE REVIEW

**Comment by TheHollidayInn** [comment]
Yea, I think it would be good to rewrite something like this using monk and other js libraries.

File: .eslintignore [hunk 1]
```diff
@@ -10,12 +10,9 @@ apidoc_build/
...
 node_modules/
 
-# Not linted
-website/client-old/
-test/client-old/spec/**/*
-
-# Temporarilly disabled. These should be removed when the linting errors are fixed TODO
+# Temporarily disabled. These should be removed when the linting errors are fixed TODO
 migrations/*
+reports/*
```
Comment (paglias) [inline .eslintignore:15]: Since it's a nw file we should have it linted before it can be merged

**Comment by lemoness** [comment]
@SabreCat What's the status of this?

**Comment by beffymaroo** [comment]
@SabreCat Is this still in process?

**Comment by SabreCat** [comment]
Oh, I'll close this for now. I think I can do some reports more properly, but haven't needed to run any in quite some time.

**Pull request closed without merging**